exonum-java-binding/core/rust/ejb-app/src/node_builder.rs

```diff
@@ -41,116 +105,101 @@ fn service_factories() -> HashMap<String, Box<ServiceFactory>> {
         TIME_SERVICE.to_owned(),
         Box::new(TimeServiceFactory) as Box<ServiceFactory>,
     );
-    service_factories.insert(
-        EJB_SERVICE.to_owned(),
-        Box::new(JavaServiceFactory) as Box<ServiceFactory>,
-    );
     service_factories
 }
 
-#[doc(hidden)]
-pub fn services_to_enable<P: AsRef<Path>>(path: P) -> HashSet<String> {
-    // Return default list if config file not found.
-    let mut services = load_enabled_services(path).unwrap_or_else(|_| {
-        let mut services = HashSet::new();
-        services.insert(CONFIGURATION_SERVICE.to_owned());
-        services
-    });
-
-    // Add EJB_SERVICE if it's missing
-    services.insert(EJB_SERVICE.to_owned());
-
-    services
-}
-
-pub fn create() -> fabric::NodeBuilder {
-    let services = services_to_enable(PATH_TO_SERVICES_TO_ENABLE);
-    let mut service_factories = service_factories();
-
-    let mut builder = fabric::NodeBuilder::new();
-    for service_name in &services {
-        match service_factories.remove(service_name) {
-            Some(factory) => {
-                builder = builder.with_service(factory);
-            }
-            None => panic!("Found unknown service name {}", service_name),
-        }
-    }
-    builder
-}
-
 #[cfg(test)]
 mod tests {
     use super::*;
     use std::io::Write;
     use tempfile::{Builder, TempPath};
 
-    fn create_config(filename: &str, cfg: &str) -> TempPath {
-        let mut cfg_file = Builder::new().prefix(filename).tempfile().unwrap();
-        writeln!(cfg_file, "{}", cfg).unwrap();
-        cfg_file.into_temp_path()
-    }
-
     #[test]
-    fn no_config() {
-        let services_to_enable = services_to_enable("");
-        assert_eq!(services_to_enable.len(), 2);
-        assert!(services_to_enable.contains(EJB_SERVICE));
-        assert!(services_to_enable.contains(CONFIGURATION_SERVICE));
+    #[should_panic(expected = "Unable to load services definition")]
```

Comment:
If that's it, such user-facing error message is not good enough.

Reply:
FYI, it's not supposed to be the full error message, but rather a substring of the message

Reply:
Actually `should_panic(expected)` checks for matching the beginning of error message. In fact, there will be smth like `Unable to load services definition: [REASON]`, where REASON could be smth like `Missing required "user_services"`